components/locale/pt_PT.tsx

```diff
@@ -31,6 +31,8 @@ const localeValues: Locale = {
     searchPlaceholder: 'Procurar...',
     itemUnit: 'item',
     itemsUnit: 'itens',
+    selectAll: 'Seleccionar Tudo',
```

Comment:
it's misspelled, its like this "Selecionar tudo", I am portuguese

Reply:
Please help to fix it.